libraries/ms-common/inc/event_queue.h

```diff
@@ -48,6 +48,8 @@ StatusCode event_raise_priority(EventPriority priority, EventId id, uint16_t dat
 // for legacy compatibility.
 #define event_raise(id, data) event_raise_priority(EVENT_PRIORITY_NORMAL, (id), (data))
 
+#define event_raise_no_data(id) event_raise((id), (0))
+
```

Comment:
Looks like the other definitions here have a descriptive comment, might wanna add one for this too.